sxt/proof/inner_product/verification_computation_gpu.h

```diff
@@ -27,8 +27,8 @@ namespace sxt::prfip {
 //--------------------------------------------------------------------------------------------------
 // async_compute_verification_exponents
 //--------------------------------------------------------------------------------------------------
-xena::future<> async_compute_verification_exponents(basct::span<s25t::element> exponents,
-                                                    basct::cspan<s25t::element> x_vector,
-                                                    const s25t::element& ap_value,
-                                                    basct::cspan<s25t::element> b_vector) noexcept;
+xena::future<> async_compute_verification_exponents2(basct::span<s25t::element> exponents,
```

Comment:
Do you want the `2` postfix on the function name?

Reply:
Fixed. Thank you.